cmd/root.go

```diff
@@ -173,20 +173,26 @@ func formatDSN(user, password, address, dbname, params string) string {
 	return mysqlConfig.FormatDSN()
 }
 
-func refresh(m *manager.Manager, update bool) {
```

Comment:
I moved this function simply because there were a bunch of functions with "refresh" in their name and it was quite confusing. I wanted to make it obvious that this function served no purpose other than wrapping the call to RefreshAll and logging the result in the context of autorefresh